pyreferrer/referrer_test.py

```diff
@@ -488,6 +488,75 @@ def test_pyreferrer_works_with_unicode_query_terms():
         'subdomain': 'buy.theanimalrescuesite',
         'tld': 'com',
         'type': 'indirect',
-        'url': 'https://buy.theanimalrescuesite.greatergood.com/products/74275-pet-lovers-ultralite-woven-mary-jane-shoes?utm_source=ARS-ARS-LAL&utm_medium=paid-fb&utm_term=02052017&utm_content=Photo&utm_campaign=PetLoversUltralite™WovenMaryJaneShoes_74275&origin=ARS_face_sponsor_ARS-LAL_PetLoversUltralite™WovenMaryJaneShoes_74275_02052017' 
+        'url': 'https://buy.theanimalrescuesite.greatergood.com/products/74275-pet-lovers-ultralite-woven-mary-jane-shoes?utm_source=ARS-ARS-LAL&utm_medium=paid-fb&utm_term=02052017&utm_content=Photo&utm_campaign=PetLoversUltralite™WovenMaryJaneShoes_74275&origin=ARS_face_sponsor_ARS-LAL_PetLoversUltralite™WovenMaryJaneShoes_74275_02052017'
```

Comment:
Nitpick: can you make this into a mocked url/parameters?

Reply:
I didn't actually add this test. Not sure why the line shows green. Not sure if I should change someone else's test? I just added from line 495 `test_different_domain_suffix_maps_to_known_label()` down. It wasn't really a necessary test but also the logic wasn't shown anywhere else so thought I'd add it.

Reply:
It looks like you removed the trailing space at the end of it, which is good